tests/web/test_pannello_admin.py

```
@@ -0,0 +1,105 @@
+            clienti[nome] = cur.lastrowid
+            c.execute('INSERT INTO access_requests(user_id) VALUES (?)', (cur.lastrowid,))
+        c.commit()
+        c.close()
+
+        dati = {'admin_cookie': main.firma_sessione(admin, 1),
+                'cliente_cookie': main.firma_sessione(clienti['ClienteUno'], 1),
+                'nome_uno': 'ClienteUno', 'nome_due': 'ClienteDue',
+                'giorni': GIORNI_CONCESSI}
+        percorso_dati = tmp_path / 'dati.json'
+        percorso_dati.write_text(json.dumps(dati), encoding='utf-8')
+        proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)